app/models/katello/concerns/organization_extensions.rb

```diff
@@ -139,9 +139,7 @@ def enabled_product_content_for(repositories)
         end
 
         def enabled_product_content
-          filtered_product_content do |pc|
-            pc.product.enabled? && pc.product.redhat?
-          end
+          Katello::ProductContent.where(:product_id => self.products.enabled.redhat)
```

Comment:
Nice

Reply:
yeah, this returned an array before which was not compatible with our scoped_search based api